### ISSUE TITLE
Check availability fails with error - CheckTeamAvailability.Run failed: { "code": "InvokerConnectionOverrideFailed", "message": "Failed to parse invoker connections from trigger 'manual' outputs. Exception: Could not find any valid connection for connection reference name 'shared_sharepointonline_1' in APIM tokens header." }

### ISSUE THREAD

**Issue by imgibason** [OfficeDev/microsoft-teams-apps-requestateam#358]
### Description

Hello All,

Second time running through this install.  Frist time was in our test tenant ;)

While on the Team Information page and when clicking the Check Availability button I get an error stating 

CheckTeamAvailability.Run failed: { "code": "InvokerConnectionOverrideFailed", "message": "Failed to parse invoker connections from trigger 'manual' outputs. Exception: Could not find any valid connection for connection reference name 'shared_sharepointonline_1' in APIM tokens header." }

I ran through the deployment guide a few more times as well as comparing setting from my test tenant but I cannot find anything wrong.

Any guidance on this would be great.

Thanks!
Ian...

### Steps to reproduce

1.
2.
3.

### Expected results

Check availability button should process correctly

### Actual Results

CheckTeamAvailability.Run failed: { "code": "InvokerConnectionOverrideFailed", "message": "Failed to parse invoker connections from trigger 'manual' outputs. Exception: Could not find any valid connection for connection reference name 'shared_sharepointonline_1' in APIM tokens header." }

### Solution component

Power App

### Operating system (environment)

Windows

**Comment by imgibason** [comment]
After a little more troubleshooting I found some errors in the CheckSiteExists Logic App.  I see it errors out on the Check if site collection exists step but more so in the Get Access Token step.  It shows an Unauthorized error.  The output shows and description of:

 ""AADSTS70002: Client application name 'c0d8a829-9ce8-45ea-93c0-e0ccc9135f34@' is not valid or the credentials used to authenticate the client could not be understood by the server.\r\nTrace ID: 31842f42-bb3c-40fe-b8b8-719f7b626d00\r\nCorrelation ID: 756395ac-20a6-4011-9f0b-1987abc36f61\r\nTimestamp: 2022-04-14 18:49:32Z","

Could it be an issue with the services account I setup in the deployment?

Thanks again for any help,
Ian...

**Comment by imgibason** [comment]
Just another Update.  I was able to get past this while working with someone else you had more experience in PowerApps.  While looking through this I noticed I had two "Check Team Availability" flows.  We ended up Editing the App -> Tree View -> Screens -> scsrnBuildNew -> btnverifyAvailability we needed to change the flow from the Action -> Power Automate button.

After saving and publishing we were able to get back the Check Availability button.

Now the only issue is its not posting a notification in the Request A Teams Admins Team we have created.  I will search the Issues thread for more information on that issue.

Hope this helps anyone else!
Ian...
